README.md

```diff
@@ -1 +1,83 @@
-2020-jun-project1-group2
\ No newline at end of file
+2020-jun-project1-group2
```

Comment:
# Migrating a Wordpress Application to AWS [2020-jun-project1-group2]


Name sounds better